### DISCUSSION TITLE
tls: first record does not look like a TLS handshake

### DISCUSSION THREAD

**Discussion by xgzlucario** [natfrp/wiki#181] [使用问题]
root@LUCARIO-PC /home [255]# ./auth-guest_linux_amd64-xxxx
===== SakuraFrp AuthPanel GuestTool =====
version 0.4 @ xxxx, 2022-08-07T08:04:37Z
请求 https://cn-cd-dx-7.natfrp.cloud:xxxx 时发生错误: Get "https://cn-cd-dx-7.natfrp.cloud:xxxx": tls: first record does not look like a TLS handshake
===== 按任意键继续 =====

连接隧道时发生该错误，这是什么问题？

**Comment by fakeboboliu** [comment]
通常来说，这说明三种可能的情况：
 1. 隧道没有在运行，需要开启隧道
 2. 访问验证已经通过，如果想要再次验证可以重启隧道
 3. 隧道关闭了访问验证，如果想要验证可以打开
